xrally_kubernetes/service.py

```diff
@@ -1117,6 +1119,157 @@ def delete_statefulset(self, name, namespace, status_wait=True):
                                    resource_type="StatefulSet",
                                    namespace=namespace)
 
+    @atomic.action_timer("kubernetes.list_node")
+    def list_node(self):
+        return self.v1_client.list_node()
+
+    def list_filtered_nodes(self, node_labels=None):
```

Comment:
Do we need 2 separate methods for listing nodes?

Reply:
i think no, we can union these methods.